gwdetchar/io/html.py

```diff
@@ -209,8 +209,8 @@ def new_bootstrap_page(base=os.path.curdir, lang='en', refresh=False,
         default: None
     """
     # get kwargs with sensible defaults
-    css = kwargs.get('css', CSS_FILES)
-    script = kwargs.get('script', JS_FILES)
+    css = CSS_FILES + kwargs.get('css', [])
+    script = JS_FILES + kwargs.get('script', [])
```

Comment:
What's the motivation of this particular change? I think the original motivation was that if the user wants to give some CSS or JS files, they should give everything they need. The original code also avoids the issue of loading multiple versions of the same library if the user wishes to select a different version of, say, jquery.

Reply:
@duncanmmacleod, I was thinking ahead to the specific use case of gwsumm, which will inherit some of these javascript and stylesheets from gwdetchar, but you're right, the proper way to do that is for _gwsumm_ to pass `css=CSS_FILES + ...` and similar for `script`.